src/dispatch.rs

```diff
@@ -444,6 +446,13 @@ pub(crate) fn gen_dispatch_str(cmd: DispatchType, dispatch: bool) -> HResult<Str
             match_mon_identifier(mon.clone())
         ),
         ToggleSpecialWorkspace => "togglespecialworkspace".to_string(),
+        RenameWorkspace(id, name) => {
+            format!(
+                "renameworkspace {} {}",
```

Comment:
You need to have `{sep}` between the args, and the dispatcher name
`dispatcher{sep}{arg} {arg2}`